channel/channel.go

```diff
@@ -8,15 +8,21 @@ import (
 
 const defaultChannelTimeout = time.Second * 1
 
-var BlockedPush = string("Push blocked on channel: ")
-var UnblockedPush = string("Push unblocked on channel: ")
-var BlockedPop = string("Pop blocked on channel: ")
-var UnblockedPop = string("Pop unblocked on channel: ")
-
 type Timed[T any] struct {
 	c chan T
 }
 
+type Error string
+
+const (
+	ErrFailedToSend    = Error("Could not send data on channel.")
+	ErrFailedToReceive = Error("Could not receive data from channel")
```

Comment:
```suggestion
	ErrFailedToSend    = Error("Could not send data on channel.")
	ErrFailedToReceive = Error("Could not receive data from channel.")
```

or

```suggestion
	ErrFailedToSend    = Error("Could not send data on channel")
	ErrFailedToReceive = Error("Could not receive data from channel")
```

for coherence 😜